config/shapes/channel_selection.py

```diff
@@ -1,38 +1,33 @@
 from ntuple_processor.utils import Selection
 
 
-def channel_selection(channel, era, special=None, vs_jet_wp="tight", vs_ele_wp="vvloose"):
+def channel_selection(channel, era, special=None, vs_jet_wp="Tight", vs_ele_wp="VVLoose"):
     # Specify general channel and era independent cuts.
     cuts = [
         ("extraelec_veto<0.5", "extraelec_veto"),
         ("extramuon_veto<0.5", "extramuon_veto"),
         ("dimuon_veto<0.5", "dilepton_veto"),
         ("q_1*q_2<0", "os"),
     ]
-    cuts = [
-        ("extraelec_veto<0.5", "extraelec_veto"),
-        ("extramuon_veto<0.5", "extramuon_veto"),
-        ("dimuon_veto<0.5", "dilepton_veto"),
-        ("q_1*q_2<0", "os"),
-    ]
-    wps_dict = {
 
-        "vvtight" : "VVTight",
-        "vtight" : "VVTight",
-        "tight" : "Tight",
-        "medium" : "Medium",
-        "loose" : "Loose",
-        "vloose" : "VLoose",
-        "vvloose" : "VVLoose",
-        "vvvloose" : "VVVLoose",
-    }
+    wps_dict = [
 
-    if vs_ele_wp not in wps_dict.keys():
+        "VVTight",
+        "VVTight",
+        "Tight",
+        "Medium",
+        "Loose",
+        "VLoose",
+        "VVLoose",
+        "VVVLoose",
+     ]
+
+    if vs_ele_wp not in wps_dict:
         print("This vs electron working point doen't exist. Please specify the correct vsEle discriminator ")
-    if vs_jet_wp not in wps_dict.keys():
-        print("This vs jet working point doen't exist. Please specify the correct vsEle discriminator ")
-    vs_ele_discr = wps_dict[vs_ele_wp]
-    vs_jet_discr = wps_dict[vs_jet_wp]
+    if vs_jet_wp not in wps_dict:
+        print("This vs jet working point doen't exist. Please specify the correct vsJet discriminator ")
+    vs_ele_discr = vs_ele_wp
```

Comment:
this redefinition is not needed, since you validate the WP anyhow, just use `vs_ele_wp` bwlog